src/ImageSharp/Metadata/Profiles/IPTC/IptcTag.cs

```diff
@@ -14,222 +14,241 @@ public enum IptcTag
         Unknown = -1,
 
         /// <summary>
-        /// Record version, not repeatable.
+        /// Record version identifying the version of the Information Interchange Model.
+        /// Not repeatable. Max length is 2.
         /// </summary>
         RecordVersion = 0,
 
         /// <summary>
-        /// Object type, not repeatable.
+        /// Object type, not repeatable. Max Length is 67.
         /// </summary>
         ObjectType = 3,
 
         /// <summary>
-        /// Object attribute.
+        /// Object attribute. Max length is 68.
         /// </summary>
         ObjectAttribute = 4,
 
         /// <summary>
-        /// Object Name, not repeatable.
+        /// Object Name, not repeatable. Max length is 64.
         /// </summary>
         Name = 5,
 
         /// <summary>
-        /// Edit status, not repeatable.
+        /// Edit status, not repeatable. Max length is 64.
         /// </summary>
         EditStatus = 7,
 
         /// <summary>
-        /// Editorial update, not repeatable.
+        /// Editorial update, not repeatable. Max length is 2.
         /// </summary>
         EditorialUpdate = 8,
 
         /// <summary>
-        /// Urgency, not repeatable.
+        /// Urgency, not repeatable. Max length is 2.
         /// </summary>
         Urgency = 10,
 
         /// <summary>
-        /// Subject Reference.
+        /// Subject Reference. Max length is 236.
         /// </summary>
         SubjectReference = 12,
 
         /// <summary>
-        /// Category, not repeatable.
+        /// Category, not repeatable. Max length is 3.
         /// </summary>
         Category = 15,
 
         /// <summary>
-        /// Supplemental categories.
+        /// Supplemental categories. Max length is 32.
         /// </summary>
         SupplementalCategories = 20,
 
         /// <summary>
-        /// Fixture identifier, not repeatable.
+        /// Fixture identifier, not repeatable. Max length is 32.
         /// </summary>
         FixtureIdentifier = 22,
 
         /// <summary>
-        /// Keywords.
+        /// Keywords. Max length is 64.
         /// </summary>
         Keywords = 25,
 
         /// <summary>
-        /// Location code.
+        /// Location code. Max length is 3.
         /// </summary>
         LocationCode = 26,
 
         /// <summary>
-        /// Location name.
+        /// Location name. Max length is 64.
         /// </summary>
         LocationName = 27,
 
         /// <summary>
-        /// Release date, not repeatable.
+        /// Release date. Format should be CCYYMMDD,
+        /// e.g. "19890317" for the 17 March 1989.
+        /// Not repeatable, max length is 8.
         /// </summary>
         ReleaseDate = 30,
 
         /// <summary>
-        /// Release time, not repeatable.
+        /// Release time. Format should be HHMMSS±HHMM,
+        /// e.g. "090000-0500" for 9 o'clock New York time (five hours behind UTC).
```

Comment:
Use `<example>` for block examples. Nest in summary to ensure they show up.

https://docs.microsoft.com/en-us/dotnet/csharp/codedoc#example